embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/api/EmbraceUrl.kt

```diff
@@ -11,6 +12,17 @@ internal class EmbraceUrl(val url: URL) {
         return EmbraceConnectionImpl(url.openConnection() as HttpURLConnection, this)
     }
 
+    fun toEndpoint(): Endpoint? {
+        return when (url.path.substringAfterLast("/")) {
+            Endpoint.EVENTS.path -> Endpoint.EVENTS
+            Endpoint.BLOBS.path -> Endpoint.BLOBS
+            Endpoint.LOGGING.path -> Endpoint.LOGGING
+            Endpoint.NETWORK.path -> Endpoint.NETWORK
+            Endpoint.SESSIONS.path -> Endpoint.SESSIONS
+            else -> null
```

Comment:
Do we want to have a default value that isn't null so if we have a ApiRequest that doesn't fit this scheme later, it'll at least fall through to a generic queue? Something like `Endpoint.Unknown`?

Reply:
Yes, that's a good idea, I added `Endpoint.Unknown`